projects/debouncer/inc/debouncer.h

```diff
@@ -0,0 +1,17 @@
+#pragma once
+// The debouncer module
+// Requires GPIO, interrupts and soft timer to be initialized
```

Comment:
I would suggest adding a high-level explanation of what problem this module solves and how it's expected to be used.